test/action_controller/json_api/linked_test.rb

```diff
@@ -82,7 +82,7 @@ def render_collection_without_include
 
           def render_collection_with_include
             setup_post
-            render json: [@post], adapter: :json_api, include: 'author, comments'
+            render json: [@post], adapter: :json_api, include: 'author,comments'
```

Comment:
:+1:

Reply:
note for future reference:

this change belongs to https://github.com/jsonapi-rb/jsonapi-renderer/pull/38 only, irrelevant of this PR. we need to make this tweak in order to make the test suite pass.